container_files/assets/www/doc/builtin/functions/EM.md

```diff
@@ -0,0 +1,40 @@
+# EM Function
+
+The Estimation-Maximisation (EM) function type takes the output dataset of the EM procedure and applies
```

Comment:
There are some inconsistency in the way maximization is written.  I would go with the american form _maximization_.